.github/workflows/ci.yml

```
@@ -0,0 +1,124 @@
+    # Frontend setup
+    - name: Install frontend dependencies
+      working-directory: ./frontend
+      run: npm ci
+
+    # Create an empty test file to prevent test runner from failing
+    - name: Create placeholder test file
+      working-directory: ./frontend
+      run: |
+        mkdir -p src/__tests__
+        echo 'import { describe, it, expect } from "vitest";
+        
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)